.github/workflows/tests.yml

```diff
@@ -0,0 +1,137 @@
+name: Lint and Test
```

Comment:
also seems old like `release.yml`

Reply:
will sort shortly by mimicing other template repos we hve

Reply:
margin as resolved as workflows were just coipoied over from not even a template repo